Description: increase address space limit
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Origin: http://askubuntu.com/questions/126625/libgcc-s-so-1-must-be-installed-for-pthread-cancel-to-work
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/vsftpd/+bug/1348972
Forwarded: via mail: Chris Evans <scarybeasts@gmail.com>
Last-Update: 2014-10-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/defs.h
===================================================================
--- trunk.orig/defs.h	2014-10-05 09:54:44.774491678 +0200
+++ trunk/defs.h	2014-10-05 09:54:44.770491596 +0200
@@ -19,7 +19,7 @@
 /* Must be at least the size of VSFTP_MAX_COMMAND_LINE, VSFTP_DIR_BUFSIZE and
    VSFTP_DATA_BUFSIZE*2 */
 #define VSFTP_PRIVSOCK_MAXSTR   VSFTP_DATA_BUFSIZE * 2
-#define VSFTP_AS_LIMIT          100UL * 1024 * 1024
+#define VSFTP_AS_LIMIT          400UL * 1024 * 1024
 
 #endif /* VSF_DEFS_H */
 
